src/main/kotlin/com/sourcegraph/cody/context/ui/AddRepositoryDialog.kt

```diff
@@ -94,7 +93,9 @@ class AddRepositoryDialog(
   }
 
   override fun doOKAction() {
-    addAction(repoUrlInputField.text.lowercase())
+    runCatching { convertGitCloneURLToCodebaseNameOrError(repoUrlInputField.text.lowercase()) }
```

Comment:
Can you move that conversion to lowercase inside `convertGitCloneURLToCodebaseNameOrError`?
That way we will never miss it... as we do now in case of the migration?